File: src/main/java/fi/finlit/edith/ui/components/AuthPanel.java
/*
 * Copyright (c) 2009 Mysema Ltd.
 * All rights reserved.
 *
 */
package fi.finlit.edith.ui.components;

import org.apache.tapestry5.ComponentResources;
import org.apache.tapestry5.ioc.annotations.Inject;

public class AuthPanel extends AuthAwarePanel{

    @Inject
    private ComponentResources resources;

    public boolean isOnPage(String page){
        return resources.getPageName().equalsIgnoreCase(page);
    }

}

File: src/main/scala/fi/finlit/edith/ui/components/AuthPanel.scala
package fi.finlit.edith.ui.components

import org.apache.tapestry5.ComponentResources
import org.apache.tapestry5.ioc.annotations.Inject
//remove if not needed
import scala.collection.JavaConversions._

class AuthPanel extends AuthAwarePanel {

  @Inject
  private var resources: ComponentResources = _

  def isOnPage(page: String): Boolean = {
    resources.getPageName.equalsIgnoreCase(page)
  }
}
